packages/integration_test/example/integration_test/_example_test_io.dart

```diff
@@ -20,26 +20,38 @@ void main() {
   final IntegrationTestWidgetsFlutterBinding binding =
       IntegrationTestWidgetsFlutterBinding.ensureInitialized()
           as IntegrationTestWidgetsFlutterBinding;
+
+  // This is required for screenshot to work.
+  binding.framePolicy = LiveTestWidgetsFlutterBindingFramePolicy.fullyLive;
+
   testWidgets('verify text', (WidgetTester tester) async {
     // Build our app and trigger a frame.
     app.main();
 
+    // Let app become fully idle.
+    await Future<void>.delayed(const Duration(seconds: 2));
     // Trace the timeline of the following operation. The timeline result will
     // be written to `build/integration_response_data.json` with the key
     // `timeline`.
-    await binding.traceAction(() async {
-      // Trigger a frame.
-      await tester.pumpAndSettle();
-
-      // Verify that platform version is retrieved.
-      expect(
-        find.byWidgetPredicate(
-          (Widget widget) =>
-              widget is Text &&
-              widget.data!.startsWith('Platform: ${Platform.operatingSystem}'),
-        ),
-        findsOneWidget,
-      );
-    });
+    // Trigger a frame.
+    await tester.pumpAndSettle();
+
+    await Future<void>.delayed(const Duration(seconds: 3));
+
+    // On Android, this is required prior to taking the screenshot.
```

Comment:
@dnfield had an opinion on this.  Happy to change it if necessary

Reply:
Changing the surface type automatically is likely to change test behavior without the developer understanding why. Particularly if the test is doing some benchmarking and the developer throws a screenshot in there.

Reply:
I see, this makes sense if the intent is to make this explicit to the user. Will this affect other `testWidget` blocks though? If so we should probably use `addTearDown` within the implementation to reset this after the current test block runs to avoid things like order-dependent tests.

Reply:
@blasten is it possible to reset this?

Reply:
yes, it's possible. I added `addTearDown` and the revert.